include/configs/piksiv3_dev.h

```diff
@@ -157,6 +157,8 @@
 
 /* Default environment */
 #define CONFIG_EXTRA_ENV_SETTINGS \
+  "pre_boot_counter=0\0" \
```

Comment:
commented in https://github.com/swift-nav/piksi_buildroot/pull/1211 as well, but maybe we should add a `uboot_env_version` or something to keep track of how we are up-revving and establishing compatibility between uboot and linux user code

Reply:
That's a much better idea than what i've been doing.